packages/data-stores/src/plans/constants.ts

```diff
@@ -17,18 +22,48 @@ interface Currency {
 	decimal: number;
 }
 
-export const plansOrder = [ PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS, PLAN_ECOMMERCE ];
+export const plansOrder = [
+	PLAN_PERSONAL,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_ECOMMERCE,
+	PLAN_PERSONAL_MONTHLY,
+	PLAN_PREMIUM_MONTHLY,
+	PLAN_BUSINESS_MONTHLY,
+	PLAN_ECOMMERCE_MONTHLY,
+];
+
+export const billedMonthlySlugs = [
+	PLAN_PERSONAL_MONTHLY,
+	PLAN_PREMIUM_MONTHLY,
+	PLAN_BUSINESS_MONTHLY,
+	PLAN_ECOMMERCE_MONTHLY,
+] as const;
 
-export type PlanPath = 'free' | 'personal' | 'premium' | 'business' | 'ecommerce';
+export const plansPaths = [
+	'free',
+	'personal',
+	'premium',
+	'business',
+	'ecommerce',
+	'personal-monthly',
+	'premium-monthly',
+	'business-monthly',
+	'ecommerce-monthly',
+] as const;
```

Comment:
I'm not sure we need the new paths. By looking at the [API response](https://public-api.wordpress.com/rest/v1.5/plans) I see that monthly plans don't have a `path_slug` attribute.